From 27941b29ab8d0184d8dd4963653796091e1524bd Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Wed, 5 Sep 2018 20:50:47 +0800
Subject: [PATCH 4577/5242] MLK-19431 ARM64: dts: fsl-imx8mm-evk: add
 off-on-delay for usdhc2

commit  c0d09ba412c14a71b7fc3f99268e4c0cba498efb from
https://source.codeaurora.org/external/imx/linux-imx.git

Make sure the time between power off and power on sd card meet the
sd SPEC, otherwise after suspend and resume, SD3.0 card will no longer
be recognized as SD3.0, but SD2.0

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
(cherry picked from commit 1ee3ea929e89fc1c8eeab129d8e64179d3dd4afa)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
index 3820856..365e2d6 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
@@ -67,6 +67,7 @@
 			regulator-min-microvolt = <3300000>;
 			regulator-max-microvolt = <3300000>;
 			gpio = <&gpio2 19 GPIO_ACTIVE_HIGH>;
+			off-on-delay = <20000>;
 			enable-active-high;
 		};
 
-- 
1.7.9.5

